packages/quantic/force-app/main/default/lwc/quanticFoldedResultList/e2e/pageObject.ts

```diff
@@ -0,0 +1,81 @@
+import type {Locator, Page, Request} from '@playwright/test';
```

Comment:
I still think we should test the folding here. If not in unit tests, I would be ok in E2E tests since it kinda makes sense to me to have folding tests in folded result lists tests. 

You could probably re-use the mocked date in the cypress tests.